packages/rum-core/src/common/config-service.js

```diff
@@ -272,6 +272,14 @@ class Config {
       storage.setItem(LOCAL_CONFIG_KEY, JSON.stringify(config))
     }
   }
+
+  dispatchEvent(name, args) {
```

Comment:
Although it might feel a bit superfluous I prefer this proxy rather than exposing private properties, but well I think it is a matter of taste in this particular context 😄